src/components/Header/index.tsx

```diff
@@ -8,19 +8,42 @@ import './index.scss';
 type HeaderProps = {
   auth: any;
   children: React.ReactNode | React.ReactNodeArray;
+  name: string;
 };
 
-export const Header = ({ auth, children }: HeaderProps) => {
+const easiLogo = (
+  <Link to="/" title="Home" aria-label="EASi home">
+    EASi
+  </Link>
+);
+
+const intakeLogo = <span aria-label="EASi home">CMS System Intake</span>;
+
+export const Header = ({ auth, children, name }: HeaderProps) => {
   const [isAuthenticated, user = {}, handleLogout] = useAuth(auth);
+  let logo;
+  if (name === 'INTAKE') {
+    logo = intakeLogo;
+  } else {
+    logo = easiLogo;
+  }
   return (
-    <header className="usa-header easi-header" role="banner">
+    <header
+      className="usa-header usa-header--extended easi-header"
+      role="banner"
+    >
       <HeaderWrapper className="usa-navbar">
         <div className="usa-logo site-logo" id="logo">
-          <em className="usa-logo__text">
-            <Link to="/" title="Home" aria-label="EASi home">
-              EASi
-            </Link>
-          </em>
+          <em className="usa-logo__text">{logo}</em>
+          {name === 'INTAKE' && (
```

Comment:
This might be the only way to handle this, but it might be better to pass the `Save & Exit` button as `children`. It is a more composable approach rather than the `Header` knowing about the different pages it's on.

Reply:
The `children` prop is already in use (by the secondary nav), I'll put the button in a different prop, unless you think it should be the other way around?